components/bl-screen-recorder-component/styles/index.less

```
@@ -0,0 +1,52 @@
+@bl-customComponent-screenRecorder-video-background: black;
+
+@bl-customComponent-screenRecorder-button-backgroundColor: @themePrimary;
+@bl-customComponent-screenRecorder-button-disabled: @disabledColor;
+@bl-customComponent-screenRecorder-button-borderRadius: @appComponentBorderRadius;
+@bl-customComponent-screenRecorder-button-color: @appTextColor;
+@bl-customComponent-screenRecorder-button-shadowColor: @appComponentShadowColor;
+@bl-customComponent-screenRecorder-button-shadowHover: 0px 2px 4px -1px fade(@bl-customComponent-screenRecorder-button-shadowColor, 20%), 0px 4px 5px 0px fade(@bl-customComponent-screenRecorder-button-shadowColor, 14%), 0px 1px 10px 0px fade(@bl-customComponent-screenRecorder-button-shadowColor, 12%);
```

```suggestion
@bl-customComponent-screenRecorder-button-shadowHover:
  0px 2px 4px -1px fade(@bl-customComponent-screenRecorder-button-shadowColor, 20%),
  0px 4px 5px 0px fade(@bl-customComponent-screenRecorder-button-shadowColor, 14%),
  0px 1px 10px 0px fade(@bl-customComponent-screenRecorder-button-shadowColor, 12%);
```

Could we do like that?